litmus-portal/frontend/src/components/Sections/Settings/AccountsTab/PersonalDetails/index.tsx

```diff
@@ -16,10 +18,16 @@ interface personaData {
 // Displays the personals details on the "accounts" tab
 const PersonalDetails: React.FC = () => {
   const classes = useStyles();
-
   const userData: UserData = useSelector((state: RootState) => state.userData);
+  const { username } = userData;
+
+  // Query to get user details
+  const { data } = useQuery<CurrentUserDetails>(GET_USER, {
```

Comment:
Can we use react state to get data?

Reply:
No because it was a part of the fix to optimize storage usage on the client, it has been purposely updated to use query from apollo client to avoid replication of data and inefficient memory usage and state management. @chandankumar4

Reply:
@chandankumar4 Apollo will serve te data from cache

Reply:
okay